lib/decompress/zstd_decompress.c

```diff
@@ -72,6 +72,140 @@
 #endif
```

Comment:
I don't see an `#include "../common/xxhash.h"` anywhere

Reply:
Hmm, `zstd_decompress.c` has `#include "../common/zstd_internal.h"`, which has `#include "xxhash.h"` - I assumed that that would be sufficient to include all the public symbols from `xxhash.h`?